src/http/methods.rs

```diff
@@ -0,0 +1,28 @@
+use std::str::FromStr;
+
+#[derive(Clone)]
+pub enum HTTPMethod {
+    GET,
+    POST,
+    PUT,
+    PATCH,
+    DELETE,
+    HEAD,
+    OPTIONS,
```

Comment:
Does it really need it's own file? :D

Reply:
[https.rs](https://github.com/adhamsalama/blitzkrieg/blob/main/src/http.rs) was crowded with many things. Frankly, I like organization and I do not see that there is a problem with a large number of files, but everything is organized.
Maybe in future you can add Http status codes enum here,  But the difficulty will be in naming the file :D